services/workflows-service/src/common/filters/AllExceptions.filter.ts

```diff
@@ -34,11 +35,14 @@ export class AllExceptionsFilter extends BaseExceptionFilter {
       .setHeader('Content-Type', 'application/json')
       .json({
         errorCode: String(HttpStatusCode[serverError.getStatus()]),
-        message: serverError.message,
-        errors,
+        message:
+          typeof serverError.getResponse() === 'string'
+            ? serverError.getResponse()
+            : serverError.message,
```

Comment:
I recall we came to an agreement `message` would be along the lines of `Bad Request` and `Validation Error` and `errors` would consist of the details.

Reply:
message always should present as it was until these days for backward compatibility which already used to pass errors to BadRequest and read a list of errors from message field,

Reply:
This has not been true for the backoffice for a while and in the collection flow error handling is limited and focused. Considering a change is needed in either of the apps making that change to handle `errors` instead of `message` makes more sense as I see it.